src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.stories.ts

```diff
@@ -0,0 +1,99 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import type { Meta, StoryObj } from "@storybook/react";
+import { SubscriberBreach } from "../../../../../../../../utils/subscriberBreaches";
+import { createRandomBreach } from "../../../../../../../../apiMocks/mockData";
+import { GuidedExperienceBreaches } from "../../../../../../../functions/server/getUserBreaches";
+import { BreachDataTypes } from "../../../../../../../functions/universal/breach";
+import { HighRiskBreachLayout } from "./HighRiskBreachLayout";
+
+const meta: Meta<typeof HighRiskBreachLayout> = {
+  title: "ResolutionLayout",
+  component: HighRiskBreachLayout,
+};
+export default meta;
+
+type Story = StoryObj<typeof HighRiskBreachLayout>;
+
+const scannedResultsArraySample: SubscriberBreach[] = Array.from(
+  { length: 5 },
+  () => createRandomBreach({ isHighRiskOnly: true })
+);
+
+//TODO: Be able to import this from getUserBreaches
+/* c8 ignore start */
+function guidedExperienceBreaches(
```

Comment:
I'd recommend in general to start functions with a verb, to ease identification and to avoid the risk of name collisions with variables (nouns), such as the one inside this very function :)

```suggestion
function getGuidedExperienceBreaches(
```